pkg/gardenadm/cmd/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package bootstrap
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+// NewCommand creates a new cobra.Command.
+func NewCommand(ioStreams genericiooptions.IOStreams) *cobra.Command {
+	opts := &Options{}
+
+	cmd := &cobra.Command{
+		Use:   "bootstrap",
+		Short: "Bootstrap the infrastructure for an Autonomous Shoot Cluster (networks, machines, etc.)",
```

Comment:
Can we make it like discover.go where the examples are not part of the short version?